build-blocks/conventions: Fill contents (#54)
open-education-hub/methodology: chapters/building-blocks/conventions/reading/read.md

# Conventions

Throughout the methodology we will use certain words and structure for uniformity and readability.
We present here these conventions.

Throughout the text, as also laid out in [the "Overview" section](../../overview/reading/read.md) we will use the words **educator** and **learner**.
We consider these terms to be generic enough to encompass the larger types of deliverers and consumers of educational content: teachers, trainers, professors, mentors, presenters on one side, and pupils, trainees, students, mentees, apprentices on the other side.

> The word **repository** is used to represent both an actual open source repository (such as those hosted on GitHub) and the actual collection of contents, disconnected from their storage environment.

Reviewer: ```suggestion
The word **repository** is used to represent both open source repositories (such as those hosted on GitHub) and the complete collection of contents, disconnected from their storage environment.
```
"actual" is an overused (and often misused) term.

> Each topic is allocated their own file / page, with liberal referencing of topics in other pages.

Reviewer: ```suggestion
Each topic is allocated its own file / page, with liberal referencing of topics on other pages.
```

This keeps things all in one place, and it makes it easy for the reader to focus only on the topic of interest.

Where possible and relevant, [roles](../../../roles/overview/reading/read.md) are used to signal who is expected to do a specific action.

> For each content type or educational activity, existing repositories are used.

Reviewer: ```suggestion
Existing repositories are used to showcase each content type or educational activity.
```

These demonstrate the practical implementation of that topic.
Existing repositories used as examples are:

- [the `operating-systems` repository](https://github.com/open-education-hub/operating-systems)
- [the `ccas` repository](https://github.com/open-education-hub/ccas)
- [the `binary-security` repository](https://github.com/open-education-hub/binary-security)

> As much as possible, concrete examples of directory structures and commands are being used.

Reviewer: ```suggestion
Concrete examples of directory structures and commands are used as much as possible.
```
